### DISCUSSION TITLE
Wait a few Seconds before Download

### DISCUSSION THREAD

**Discussion by lumentae** [DirectoryLister/DirectoryLister#677] [Help & Support]
Hey, I'm new to DirectoryLister, and was asking if there is a function to wait a few seconds before the download starts

Would be nice if it has this feature

**Comment by PHLAK** [comment]
This is not currently possible and don't understand why you would want this feature. Can you please elaborate on how this would be useful?

**Comment by lumentae** [comment]
Basically, I dont want people to overflow my servers somehow using the downloads
That's why

**Reply by PHLAK** [reply]
I see. The generated zip archives are cached for a short time to reduce load on the server but, as previously stated, there's not currently a way to add an arbitrary time limit before a download starts.
